analytics/views.py

```diff
@@ -1728,6 +1754,9 @@ def by_used_time(row: Dict[str, Any]) -> str:
         "Android",
     ]
 
+    if is_bot:
+        cols[2] = "Bot Owner"
```

Comment:
it might be better to have the column be `"Role" if not is_bot else "Bot owner"`.

Reply:
Tangential, but the last 4 columns are not really useful on the bots page.  It'd be nice to remove them, possibly as a follow-up diff.